website_img_bg_style/static/src/css/styles.css

```diff
@@ -0,0 +1,357 @@
+@charset "UTF-8";
```

Comment:
Don't include this file and map one, as version 9 now generates it on the fly from sass file. Map file isn't generated in fact, but we can't maintain a possible obsolete copy of it.